scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,442 @@
+# -*- coding: utf-8 -*-
+"""
+This script computes four myelin indices maps from the Magnetization Transfer
```

Comment:
Somewhere in there, maybe mention that all input must have a matching json with the same filename.
And do you recommend using dcm2niix to get the right format for the json (recommend the software you want people to use)